src/App.tsx

```diff
@@ -1,61 +1,125 @@
+import { useState } from 'react';
 import './App.scss';
+// import classNames from 'classnames';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, TodoUser, User } from './interfaces';
+
+const todosUsersFiltered: TodoUser[] = todosFromServer.map((todo: Todo) => {
+  const matchUser = usersFromServer.find(
+    (user: User) => user.id === todo.userId,
+  );
+
+  return {
+    ...todo,
+    user: matchUser,
+  };
+});
 
 export const App = () => {
+  const [count, setCount] = useState(16);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
```

Comment:
user can be 0
maybe user id?
but we never have id = 0

Reply:
I can add a placeholder, but in the video for this task, Misha showed exactly this method (when the index 0 stores a default value that cannot be selected)